src/bar-chart/bar.component.ts

```diff
@@ -155,6 +157,12 @@ export class BarComponent implements OnChanges {
     return path;
   }
 
+  hideBar() {
```

Comment:
We should cache this value on the component and update it in the `update` method. Calling functions in the template is not performant, as it will be reevaluated over and over.

Reply:
633d5cc addressed it